specification/resources/resource-manager/Microsoft.Solutions/stable/2019-07-01/managedapplications.json

```diff
@@ -1683,6 +1870,77 @@
       ],
       "description": "The managed application provider authorization."
     },
+    "ApplicationPackageContact": {
+      "properties": {
+        "contactName": {
+          "type": "string",
+          "description": "The contact name."
+        },
+        "email": {
+          "type": "string",
+          "description": "The contact email."
+        },
+        "phone": {
+          "type": "string",
+          "description": "The contact phone number."
+        }
+      },
+      "required": [
+        "email",
+        "phone"
+      ],
+      "description": "The application package contact information."
+    },
+    "ApplicationPackageSupportUrls": {
+      "properties": {
+        "publicAzure": {
+          "type": "string",
+          "description": "The public azure support URL."
+        },
+        "governmentCloud": {
+          "type": "string",
+          "description": "The government cloud support URL."
+        }
+      },
+      "description": "The appliance package support URLs."
+    },
+    "ApplicationBillingDetailsDefinition": {
+      "properties": {
+        "resourceUsageId": {
+          "type": "string",
+          "description": "The managed application resource usage Id."
+        }
+      },
+      "description": "Managed application billing details definition."
+    },
+    "ApplicationJitAccessPolicy": {
+      "properties": {
+        "JITAccessEnabled": {
+          "type": "string",
+          "description": "Whether the JIT access is enabled."
+        },
+        "JITApprovalMode": {
+          "$ref": "#/definitions/JitApprovalMode",
+          "description": "JIT approval mode."
+        },
+        "JITApprovers": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/JitApproverDefinition"
+          },
+          "description": "The JIT approvers"
+        },
+        "maximumJitAccessDuration": {
+          "readOnly": true,
+          "type": "string",
```

Comment:
A string?  What format is this string?  (please add this info in the description)